src/langchain_google_alloydb_pg/engine.py

```diff
@@ -764,6 +767,80 @@ def init_document_table(
             )
         )
 
+    async def _ainit_checkpoint_table(
+        self, schema_name: str = "public"
+    ) -> None:
+        """
+        Create AlloyDB tables to save checkpoints.
+
+        Args:
+            schema_name (str): The schema name to store the checkpoint tables.
+                Default: "public".
+
+        Returns:
+            None
+        """
+        create_checkpoints_table = f"""CREATE TABLE IF NOT EXISTS "{schema_name}".{CHECKPOINTS_TABLE}(
+            thread_id TEXT NOT NULL,
+            checkpoint_ns TEXT NOT NULL DEFAULT '',
+            checkpoint_id TEXT NOT NULL,
+            parent_checkpoint_id TEXT,
+            checkpoint JSONB NOT NULL,
+            metadata JSONB NOT NULL DEFAULT '{{}}',
+            channel TEXT NOT NULL,
+            version TEXT NOT NULL,
```

Comment:
In the design doc, we don't have these specified. Is there a reason why we have these here?